zulipterminal/ui_tools/boxes.py

```diff
@@ -1726,7 +1726,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             self.panel_view.keypress(size, "esc")
         elif is_command_key("ENTER", key) and not self.panel_view.empty_search:
             self.panel_view.view.controller.exit_editor_mode()
-            self.set_caption([("filter_results", "Search Results"), " "])
+            self.set_caption([("filter_results", " Search Results"), " "])
```

Comment:
I think adding " " as another item would look better as this currently applies the styling to `Search results` as well. :)

Reply:
I definitely agree with this. Generally the extra space looks good though :+1:

Reply:
I think it look weird that way, having spaces on both sides is another option which looks better. (See below)
<img src='https://user-images.githubusercontent.com/64144419/124892068-28d69600-dff7-11eb-9ad6-63eda6d3fe76.png' height=300>
<img src='https://user-images.githubusercontent.com/64144419/124891894-017fc900-dff7-11eb-9282-1bdeb5bfb5f7.png' height=270>